### PR TITLE
refactor: Format the entire codebase using black

### PR DESCRIPTION
**Pull request by dhruvmanila**
Re-opening the PR #147 

Subtask of https://github.com/internetarchive/openlibrary/issues/4776

This pull request was created with the command: `black --skip-string-normalization .`

**NOTE:**
It is a good practice to ignore such large changes done by autoformatters for `git blame` by storing the commit hash in a separate file like `.git-blame-ignore-revs`. I will open a separate PR after this is merged as we need to commit hash.

### CODE REVIEW

**Comment by dhruvmanila** [comment]
Sorry, the editor ran `isort` as well so I had to force push for just formatting through `black`. The error is from `flake8` for [these long lines](https://github.com/internetarchive/infogami/blob/master/test/test_dbstore.py#L50-L55).

**Comment by cclauss** [comment]
Those long lines are to support the difference between Python 2 and Python 3.

Given that this codebase no longer supports legacy Python, could this code be modified to support Python 3 only with lines that are black-compliant?

**Comment by dhruvmanila** [comment]
So, the dictionary are error strings where the keys are the ones for Python 2 and the values are for Python 3.

Alright, this seems simple. Let me try it out.

**Comment by dhruvmanila** [comment]
I am thinking that the manual changes made to remove the Python 2 test cases should not be done in this PR as this should only contain the auto-formatting part. I am removing those changes and I will open another PR for that.

It will be easy for `git blame` and also looking at the commit history.

File: infogami/core/code.py [hunk 1]
```diff
@@ -351,26 +385,28 @@ def POST(self):
...
     """Lists all pages with name path/*"""
+
     def GET(self):
         i = web.input("type", property="key")
-        q = {
-            i.property + '~': i.q + '*',
-            'type': i.type,
-            'limit': int(i.limit)
-        }
+        q = {i.property + '~': i.q + '*', 'type': i.type, 'limit': int(i.limit)}
```
Comment (hornc) [inline infogami/core/code.py:397]: This change seems less clear than the original. I'm surprised this is joining dicts like this.

Comment (dhruvmanila) [reply infogami/core/code.py:397]: `black` wants to keep everything in one line unless that exceeds the character limit. In cases like this, we can force `black` to keep the entries on separate line by adding a trailing comma.

Should I add it?

File: infogami/infobase/_dbstore/sequence.py [hunk 2]
```diff
@@ -14,16 +15,20 @@ def fire_event(self, event_name, name, value):
 
     def get_value(self, name):
         try:
-            return self.db.query("SELECT * FROM seq WHERE name=$name", vars=locals())[0].value
+            return self.db.query("SELECT * FROM seq WHERE name=$name", vars=locals())[
+                0
+            ].value
```
Comment (hornc) [inline infogami/infobase/_dbstore/sequence.py:20]: This one is just funny 😂

**Comment by dhruvmanila** [comment]
Rebased onto the latest master solving the merge conflicts.

File: infogami/core/code.py (hunk 1 above)
Comment (cclauss) [reply infogami/core/code.py:397]: # fmt: off 
_code that I do not want reformatted_
# fmt: on

File: infogami/infobase/_dbstore/sequence.py (hunk 2 above)
Comment (cclauss) [reply infogami/infobase/_dbstore/sequence.py:20]: # fmt: off 
_code that I do not want reformatted_
# fmt: on

File: infogami/core/code.py (hunk 1 above)
Comment (dhruvmanila) [reply infogami/core/code.py:397]: Actually, it is a good practice to keep a trailing comma to have a sane git diff. There's an entire plugin just to add that comma and a flake8 plugin as well to lint files for that comma.
Formatting plugin: https://github.com/asottile/add-trailing-comma
flake8 plugin: https://github.com/PyCQA/flake8-commas

Comment (cclauss) [reply infogami/core/code.py:397]: The very first line of the psf/blacks README says:
> Black is the uncompromising Python code formatter. By using it, you agree to cede control over minutiae of hand-formatting. In return, Black gives you speed, determinism, and freedom from pycodestyle nagging about formatting. You will save time and mental energy for more important matters.

I tend to want to _cede control over minutiae_ but...

**Review by cclauss — APPROVED** [review]
LGTM

File: infogami/core/code.py (hunk 1 above)
Comment (dhruvmanila) [reply infogami/core/code.py:397]: At the end, it's only about consistency with whichever style we choose.

**Pull request merged**